Common/Util/LeanData.cs

```diff
@@ -1082,20 +1082,39 @@ public static bool TryParsePath(string fileName, out Symbol symbol, out DateTime
                     // Gather components used to create the security
                     market = info[startIndex + 1];
                     ticker = info[startIndex + 3];
-
-                    // If resolution is Daily or Hour, we do not need to set the date and tick type
-                    if (resolution < Resolution.Hour)
+                    
+                    // Remove the ticktype from the ticker 
+                    // TODO: This seems very poor, addresses hour and daily resolution paths where ticker is only in the zip file name.
+                    if (securityType == SecurityType.Crypto || securityType == SecurityType.Future)
```

Comment:
Thinking generically we might not really need this `if` check? if ticker contains `'_'` we know the ticker is the first section always?